### PR TITLE
Create customer page added

### PR DESCRIPTION
**Pull request by manic1990**
Added create customer page
Added customer list page
Added Authorize

### CODE REVIEW

File: BookingSystem.DAL/Repository/AttributeRepository.cs [hunk 1]
```diff
@@ -23,27 +18,27 @@ public AttributesVM saveAttribute(AttributesVM model)
...
-            attributes.nightly_average_price_cents = model.nightly_average_price_cents;
-            attributes.deposit_price_cents = model.deposit_price_cents;
-            attributes.balance_price_cents = model.balance_price_cents;
-            attributes.cancellable_until = model.cancellable_until;
-            attributes.cancelled = model.cancelled;
-            attributes.booking_date = model.booking_date;
-            attributes.accompany_guests_consent = model.accompany_guests_consent;
-            attributes.failure_reason = model.failure_reason;
-            attributes.currency_code = model.currency_code;
+            attributes.reference_number = model.ReferenceNumber;
```
Comment (vivekr2021) [inline BookingSystem.DAL/Repository/AttributeRepository.cs:21]: Why did the casing are changed?

File: BookingSystem.DAL/Repository/BookingRepository.cs [hunk 2]
```diff
@@ -37,17 +38,17 @@ public List<BookingDataVM> GetBookingDetails()
...
-                                       reference_number = attr.reference_number,
+                                       ReferenceNumber = attr.reference_number,
                                        Customers = new CustomerVM
                                        {
                                            CustomerID = c.CustomerID,
-                                           first_name = c.first_name,
-                                           last_name = c.last_name,
-                                           email = c.email,
-                                           phone_number = c.phone_number,
+                                           FirstName = c.first_name,
```
Comment (vivekr2021) [inline BookingSystem.DAL/Repository/BookingRepository.cs:45]: Why do the casing is changed?

File: BookingSystem.DAL/Repository/AttributeRepository.cs (hunk 1 above)
Comment (manic1990) [reply BookingSystem.DAL/Repository/AttributeRepository.cs]: removed model changes

File: BookingSystem.DAL/Repository/BookingRepository.cs (hunk 2 above)
Comment (manic1990) [reply BookingSystem.DAL/Repository/BookingRepository.cs]: removed model changes

**Pull request merged**